### PR TITLE
Webpack dashboard view code

### PR DESCRIPTION
**Pull request by jrbotros**

### CODE REVIEW

File: orchestra/static/orchestra/config.es6.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import dashboardTemplate from 'orchestra/dashboard/partials/dashboard.html'
 import taskTemplate from 'orchestra/task/task.html'
+import dashboardTemplate from 'orchestra/dashboard/dashboard.html'
```
Comment (marcua) [inline orchestra/static/orchestra/config.es6.js:1]: keep on line 1 for alpha ordering

File: orchestra/static/orchestra/dashboard/dashboard.controller.es6.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import './dashboard.scss'
+
+export default function DashboardController (
+      $scope, $http, $location, $timeout, orchestraTasks) {
+  'ngAnnotate'
+  var vm = this
+
+    // Surface service to interpolator
```
Comment (marcua) [inline orchestra/static/orchestra/dashboard/dashboard.controller.es6.js]: some of the spacing in this file looks off

File: orchestra/static/orchestra/main.es6.js [hunk 3]
```diff
@@ -6,20 +6,20 @@ import {
 
 import common from 'orchestra/common/common.module.es6.js'
 import task from 'orchestra/task/task.module.es6.js'
+import dashboard from 'orchestra/dashboard/dashboard.module.es6.js'
```
Comment (marcua) [inline orchestra/static/orchestra/main.es6.js:8]: alpha ordering :)

**Comment by marcua** [comment]
lgtm after nitpicks

File: orchestra/static/orchestra/config.es6.js (hunk 1 above)
Comment (jrbotros) [reply orchestra/static/orchestra/config.es6.js:1]: ✅

File: orchestra/static/orchestra/dashboard/dashboard.controller.es6.js (hunk 2 above)
Comment (jrbotros) [reply orchestra/static/orchestra/dashboard/dashboard.controller.es6.js]: ✅

File: orchestra/static/orchestra/main.es6.js (hunk 3 above)
Comment (jrbotros) [reply orchestra/static/orchestra/main.es6.js:8]: ✅

**Pull request merged**